presto-main/src/main/java/com/facebook/presto/operator/scalar/SessionInformationFunctions.java

```diff
@@ -23,9 +23,9 @@
 import static com.facebook.presto.spi.function.SqlFunctionVisibility.HIDDEN;
 import static io.airlift.slice.Slices.utf8Slice;
 
-public final class SessionFunctions
+public final class SessionInformationFunctions
```

Comment:
I think it is clearer to call it SessionInformationFunction because it is not a function defined in a session. The function definition is the same no matter what session, and it retrieves a specific property of a session.

Reply:
Ya, but functions defined in a session will never end up in the source code. So there should be no confusion that these functions are not defined in the session?

Reply:
Ok, I've changed it back to SessionFunctions.